tests/phpunit/CRM/Financial/BAO/FinancialAccountTest.php

```diff
@@ -249,9 +249,9 @@ public function testcheckFinancialTypeHasDeferred() {
     $cid = $this->individualCreate();
     $params = array(
       'contact_id' => $cid,
-      'receive_date' => '2010-01-20',
+      'receive_date' => '2016-01-20',
       'total_amount' => 100,
-      'financial_type_id' => 3,
+      'financial_type_id' => 4,
```

Comment:
FYI: So it appears that this is Event Fee financial type (https://github.com/civicrm/civicrm-core/blob/master/xml/templates/civicrm_data.tpl#L128) which has a deferred revenue account (https://github.com/civicrm/civicrm-core/blob/master/xml/templates/civicrm_data.tpl#L889) defined for it at 
https://github.com/civicrm/civicrm-core/blob/master/xml/templates/civicrm_data.tpl#L1700 so the created contribution should have a financial type with a deferred revenue account.